{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Obtain information about directories for visualization.\n",
    "\n",
    "File 1:\n",
    "    One column with all nodes. Extended rows with all children of parent nodes\n",
    "\n",
    "File 2:\n",
    "    Columns: File name, size, extension. All files and directories. Directories have empty size and extension values."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "os.chdir('/home/ubuntu/IBD-Network-Models-Reproduction/')\n",
    "startpath = 'PAPER_DATA'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PAPER_DATA/\n",
      "    Gene.expression.by.SNP.genotype/\n",
      "        IBD.GPS_SNP_HLA.subtypes_coefficients_with_risk_allele_frequencies.xlsx\n",
      "        score_density.png\n",
      "        CERTIFI/\n",
      "            genotype_EAF.xls\n",
      "            stats.xls\n",
      "        MSH/\n",
      "            genotype_EAF.xls\n",
      "            stats.xls\n",
      "        RISK/\n",
      "            genotype_EAF.xls\n",
      "            stats.xls\n",
      "    IBD Networks/\n",
      "        Co-expressionnetworks/\n",
      "            Celltype_Enrichment/\n",
      "                intersect_MSH2_ileum_filtered-vs-abc-cell-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_RISK-filtered-vs-abc-tis-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_MSH2_colon_filtered-vs-abc-cell-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_RISK-filtered-vs-abc-cell-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_ALL_MSH2_filtered-vs-abc-tis-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_T26-vs-abc-tis-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_MSH2_colon_filtered-vs-abc-cell-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_MSH2_ileum_filtered-vs-abc-tis-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_T26-vs-abc-cell-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_RISK-filtered-vs-abc-tis-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_MSH2_colon_filtered-vs-abc-tis-gpl570-signatures-filtered_pair-restricted.xls\n",
      "                intersect_MSH2_ileum_filtered-vs-abc-cell-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_T26-vs-abc-tis-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_MSH2_colon_filtered-vs-abc-tis-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_MSH2_ileum_filtered-vs-abc-tis-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_ALL_MSH2_filtered-vs-abc-tis-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_T26-vs-abc-cell-gpl570-signatures-lowent_pair-restricted.xls\n",
      "                intersect_RISK-filtered-vs-abc-cell-gpl570-signatures-lowent_pair-restricted.xls\n",
      "            Release Modules/\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-ileum_imgHierModLabeled.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex-Rgn_ALL_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-ileum_imgHierClust.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-colon_imgHierClust.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_UC-colon_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex-Rgn_ALL_imgHeatmap.png\n",
      "                IBD_risk_FPKMexpressionValues_filtered_imgHierModLabeled.png\n",
      "                IBD_risk_FPKMexpressionValues_filtered_imgHierModules.png\n",
      "                IBD_T26_expression_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-colon_imgHierModLabeled.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-colon_imgHierClust.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-colon_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-ileum_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-ileum_imgHierModLabeled.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-ileum_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-colon_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_UC-colon_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_UC-colon_imgHierClust.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex-Rgn_ALL_imgHierClust.png\n",
      "                IBD_risk_FPKMexpressionValues_filtered_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-ileum_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-colon_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex-Rgn_ALL_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_UC-colon_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-colon_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-colon_imgHierModLabeled.png\n",
      "                IBD_risk_FPKMexpressionValues_filtered_imgHeatmap.png\n",
      "                IBD_T26_expression_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-ileum_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-colon_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex-Rgn_ALL_imgHierModLabeled.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-ileum_tommodules.xls\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-ileum_imgHierClust.png\n",
      "                IBD_T26_expression_imgHierModLabeled.png\n",
      "                IBD_T26_expression_imgHeatmap.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_UC-colon_imgHierModLabeled.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CL-ileum_imgHierModules.png\n",
      "                msbb-ibd_ucsc-clean-normed_GeneWManyZerosRemoved_Adj-Age-Sex_CD-colon_imgHierModules.png\n",
      "            MSigDB_Enrichment/\n",
      "                IBD_risk_processed-expression-dataT_tommodules_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_filtered_tommodules_Ontology.xls\n",
      "                IBD_T26_expression_tommodules2_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_ileum_filtered_tommodules_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_colon_filtered_tommodules_Ontology.xls\n",
      "            Ontology Analysis/\n",
      "                IBD_MSH2_ucsc-Clean_ileum_filtered_tommodules_OntologyTop10-sortByModule.xls\n",
      "                IBD_MSH2_ucsc-Clean_colon_filtered_tommodules_OntologyTop10-sortByModule.xls\n",
      "                IBD_MSH2_ucsc-Clean_filtered_tommodules_OntologyTop10-sortByModule.xls\n",
      "                IBD_T26_expression_tommodules2_OntologyTop10-sortByModule.xls\n",
      "                IBD_risk_FPKMexpressionValues_filtered_tommodules_OntologyTop10-sortByModule.xls\n",
      "            GO_TermEnrichment/\n",
      "                IBD_risk_processed-expression-dataT_tommodules_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_filtered_tommodules_Ontology.xls\n",
      "                IBD_T26_expression_tommodules2_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_ileum_filtered_tommodules_Ontology.xls\n",
      "                IBD_MSH2_ucsc-Clean_colon_filtered_tommodules_Ontology.xls\n",
      "            IBD_T26_expression_Module-Trait-Correlations/\n",
      "                IBD_T26_expression_tommodules_aPC-GS-pval.xls\n",
      "                IBD_T26_expression_tommodules_aPC-GS.xls\n",
      "                IBD_T26_expression_tommodules_aPC-GS-fdr.xls\n",
      "        Bayesiannetworks/\n",
      "            revisedpediatricnetwork.cys\n",
      "            originalimmunenetwork.cys\n",
      "            revisedMSHadvancednetwork.cys\n",
      "            revisedCERTIFImodantiTNFrefractory.cys\n",
      "    IBD Datasets/\n",
      "        Gene expression by SNP genotype/\n",
      "            IBD_PGS_based_on_SNPs_for_RISK_and_CERTIFI_studies/\n",
      "        Macrophagesignatures/\n",
      "            MSG_signature.txt\n",
      "            MSS_signature.txt\n",
      "        IBDclinicalcorrelationDEsignatures/\n",
      "            CERTIFI-Lactoferrin-.2+.2.txt\n",
      "            CERTIFI-FCal.txt\n",
      "            MSSMintestinepathscorecorr.txt\n",
      "            CERTIFI-Lactoferrin2.txt\n",
      "            MSH-UCvsCLcolon-DE.txt\n",
      "            CERTIFI-Diseaseduration.txt\n",
      "            CERTIFI-CRP.txt\n",
      "            CERTIFI-FCal-.3+.3.txt\n",
      "            MSH-CDvsCLcolon-DE.txt\n",
      "            MSH-CDvsUCcolon-DE.txt\n",
      "    CERTIFI DNA/\n",
      "        plink.fam\n",
      "        plink.bed\n",
      "        plink.bim\n",
      "    Code/\n",
      "        enrichment/\n",
      "        PatientDEsignatures/\n",
      "            ChipAnnotation.RData\n",
      "            DiffExpAnalysisV2.R\n",
      "            ProcFxns.R\n",
      "            BiopsyT26geneExp_wk0.RData\n",
      "            DiffExpAnalysisV2.RData\n",
      "            SummarizeResults20160309.R\n",
      "        MouseDEsignatures/\n",
      "            gene_raw_counts_matrix2\n",
      "            ensmusg_genesymbol\n",
      "        eQTL/\n",
      "            chr14.dose.txt.gz\n",
      "            gt/\n",
      "                step.5.impute.dose.matrix/\n",
      "                    chr16.dose.txt.gz\n",
      "                    chr8.dose.txt.gz\n",
      "                    chr2.dose.txt.gz\n",
      "                    chr18.dose.txt.gz\n",
      "                    chr9.dose.txt.gz\n",
      "                    chr19.dose.txt.gz\n",
      "                    chr10.dose.txt.gz\n",
      "                    chr13.dose.txt.gz\n",
      "                    chr7.dose.txt.gz\n",
      "                    chr22.dose.txt.gz\n",
      "                    chr17.dose.txt.gz\n",
      "                    chr6.dose.txt.gz\n",
      "                    chr12.dose.txt.gz\n",
      "                    chr21.dose.txt.gz\n",
      "                    chr15.dose.txt.gz\n",
      "                    chr20.dose.txt.gz\n",
      "                    chr1.dose.txt.gz\n",
      "                    chr14.dose.txt.gz\n",
      "                    chr4.dose.txt.gz\n",
      "                    chr11.dose.txt.gz\n",
      "                    chr3.dose.txt.gz\n",
      "                    chr5.dose.txt.gz\n",
      "                eQTL-tools.imputed.SNP.anno/\n",
      "                    snpspos.chr2.txt.gz\n",
      "                    snpspos.chr8.txt.gz\n",
      "                    snpspos.chr7.txt.gz\n",
      "                    snpspos.chr20.txt.gz\n",
      "                    snpspos.chr14.txt.gz\n",
      "                    snpspos.chr12.txt.gz\n",
      "                    snpspos.chr19.txt.gz\n",
      "                    snpspos.chr5.txt.gz\n",
      "                    snpspos.chr4.txt.gz\n",
      "                    snpspos.chr22.txt.gz\n",
      "                    snpspos.chr21.txt.gz\n",
      "                    snpspos.chr13.txt.gz\n",
      "                    snpspos.chr1.txt.gz\n",
      "                    snpspos.chr17.txt.gz\n",
      "                    snpspos.chr10.txt.gz\n",
      "                    snpspos.chr6.txt.gz\n",
      "                    snpspos.chr3.txt.gz\n",
      "                    snpspos.chr11.txt.gz\n",
      "                    snpspos.chr16.txt.gz\n",
      "                    snpspos.chr15.txt.gz\n",
      "                    snpspos.chr9.txt.gz\n",
      "                    snpspos.chr18.txt.gz\n",
      "            adj.gex/\n",
      "                obs/\n",
      "                    ucsc.10PC.adj.txt\n",
      "            code/\n",
      "                eqtl.tools-0.0.28.tar.gz\n",
      "                step.2.merge.peaks.pl\n",
      "                step.3.fdr.pl\n",
      "                step.1.1.pickup.sub.eqtl-lm-peaks.pl\n",
      "                step.4.cut.peak.pl\n",
      "                eqtl.tools-0.0.28/\n",
      "                    COPYING\n",
      "                    generate-example-data\n",
      "                    kruskal\n",
      "                    VERSION\n",
      "                    INSTALL\n",
      "                    table-subset\n",
      "                    split-rows\n",
      "                    Makefile\n",
      "                    lm-sex\n",
      "                    eqtl-sex-peaks\n",
      "                    lm-cis\n",
      "                    lm-suff\n",
      "                    lm-fdr\n",
      "                    lm-sort\n",
      "                    find-peaks\n",
      "                    eqtl-lm-peaks\n",
      "                    cov_pairwise.c\n",
      "                    examples/\n",
      "                        README\n",
      "                    test/\n",
      "                        peaksTestInputAnnotation.tab\n",
      "                        peaksTestInputData.tab\n",
      "                        X.small.tab\n",
      "                        X.ann\n",
      "                        femalesIDs\n",
      "                        Y.small.tab\n",
      "                        Y.ann\n",
      "                        malesIDs\n",
      "                        Y.tab\n",
      "                        X.tab\n",
      "            input dataset/\n",
      "                Ucsc.annotation.hg19.txt\n",
      "        Macrophage/\n",
      "            NCKAP1L.R\n",
      "            TNFAIP3_code.R\n",
      "            New_group_May_11_2015.R\n",
      "            GPR65.R\n",
      "            Janssen_R_code_T17_T26.R\n",
      "            ReadMeForPhnotypeCorrelation.R\n",
      "            KD_gene_expression.Rdata\n",
      "            Venn.R\n",
      "            FRP1_code_updated.R\n",
      "            Macrophage_total_code.R\n",
      "            Biopsy.R\n",
      "            LAPTM5.R\n",
      "            example.R\n",
      "            MAFB_code_updated.R\n",
      "            GBP5_code.R\n",
      "            DOK3.R\n",
      "            GBP5.R\n",
      "            LAPTM5_code_updated.R\n",
      "            GPMS3.R\n",
      "            NCKAP1L_code.R\n",
      "            DOK3_code_updated.R\n",
      "            FPR1.R\n",
      "            Total_R.R\n",
      "            GPR65_code.R\n",
      "            R_key.R\n",
      "            NewR.R\n",
      "            Macrophage_0419_2015.R\n",
      "            SLAMF1_code_updated.R\n",
      "            AIF1_code.R\n",
      "            R04262015updated.R\n",
      "            DOCK3.R\n",
      "            workingCode.R\n",
      "            Checking_Code.R\n",
      "            MAFB_code.R\n",
      "            MAFB.R\n",
      "            JanssenKO_1.R\n",
      "            Mapping_aa.R\n",
      "            GPSM3_code_updated.R\n",
      "            TNFAIP3_code_updated.R\n",
      "            GBP5_code_updated.R\n",
      "            TNFAIP3.R\n",
      "            SLAMF1.R\n",
      "            summarySE.R\n",
      "            GPR65_code_updated.R\n",
      "            AIF1_code_updated.R\n",
      "            AIF1.R\n",
      "            LAPTM5_code.R\n",
      "            GPSM3_code.R\n",
      "            Correlation_code.R\n",
      "            New_R_code_May_8_2015.R\n",
      "            NCKAP1L_code_updated.R\n",
      "            DOK3_code.R\n",
      "            SLAMF1_code.R\n",
      "            Blood_Correlation2.R\n",
      "            Summary.R\n",
      "            R04262015.R\n",
      "            FPR1_code.R\n",
      "        GWASmapandIAMKDG/\n",
      "            KDG pipeline in steps/\n",
      "                step 3 (project CIC IBD networks derived from core IAM onto intestinal BNs to identify key drivers).R\n",
      "                step 4.2 (computing overlaps between the KDG KO signatures and mac RNAi KD signatures and the mac specific cell signatures).R\n",
      "                step 4.4 (identify genes in the IBD networks that are least likely to be KDGs).R\n",
      "                step 4 (final ranking).R\n",
      "                step 5 (form the macrophage component of the IBD network).R\n",
      "                step 4.3 (compute mean rank per trait across IBD networks).R\n",
      "                step 1 generate core IAM module.R\n",
      "                step 4.1 (modifying core IAM based on final ranking to recompute starting with step 2).R\n",
      "                step 1b (project IBD signatures onto coexpression networks to assess enrichments of tagged modules).R\n",
      "                step 2 (project core -- combo -- IAM module onto intestinal BNs to get IBD CIC networks).R\n",
      "            GWASmapandenrich/\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps (CD4 T Cell Set).R\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps (Myeloid Set).R\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps (Digestive Set).R\n",
      "                CD4Generate overlap between cres-gwas overlaps and eqtl genes.R\n",
      "                NKcellsgenerate overlap between cres-gwas overlaps and eqtl genes.R\n",
      "                myeloid generate overlap between cres-gwas overlaps and eqtl genes.R\n",
      "                generate overlap between cres-gwas overlaps and eqtl genes.Bcell.R\n",
      "                link eqtl and gwas snps to ibd genes (1).R\n",
      "                CD8Tcell.generate overlap between cres-gwas overlaps and eqtl genes.R\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps (CD8 T Cell Set).R\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps NK Cell Set).R\n",
      "                Assemble CRE and GWAS sets of interest and generate overlaps (B Cell Set).R\n",
      "                Digestive-Assemble CRE and GWAS sets of interest and generate overlaps (Digestive Set).R\n",
      "        CREenrich/\n",
      "            CES.R.txt\n",
      "            db.Rdata\n",
      "            .Rhistory\n",
      "            CRE.tsv\n",
      "        TF/\n",
      "            main_TFA.R\n",
      "            Run_InferTFA_Public.R\n",
      "            GetOptimalWindowandActivity_Core_Public.R\n",
      "        analysis/\n",
      "        Network/\n",
      "            codeforenrichmenttable.R\n",
      "            co-expression network/\n",
      "                RUN_KDA.R\n",
      "                Curate_Signature.R\n",
      "                DEG.curated.RData\n",
      "                R_functions.R\n"
     ]
    }
   ],
   "source": [
    "# Function to print directory with indents\n",
    "def list_files(startpath):\n",
    "    for root, dirs, files in os.walk(startpath):\n",
    "        level = root.replace(startpath, '').count(os.sep)\n",
    "        indent = ' ' * 4 * (level)\n",
    "        print('{}{}/'.format(indent, os.path.basename(root)))\n",
    "        subindent = ' ' * 4 * (level + 1)\n",
    "        for f in files:\n",
    "            print('{}{}'.format(subindent, f))\n",
    "            \n",
    "list_files(startpath)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "# File 1: Make dictionary of parents as keys with values of list of children\n",
    "directories = {}\n",
    "for root, dirs, files in os.walk(startpath):\n",
    "    parent = root # The root is the parent node\n",
    "    children = [root+'/'+x for x in (dirs + files)] # All files and directories are children\n",
    "    directories[parent] = children # Add to dictionary: Parent = key, Children = values\n",
    "    directories.update(dict(zip([root+'/'+x for x in files], [[]]*len(files)))) \n",
    "    # Add files as nodes with no children"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Write file 1 to csv\n",
    "import csv\n",
    "filename = 'DirectoryNodes_wPaths.csv'\n",
    "\n",
    "with open(filename, 'w', newline='') as csvfile:\n",
    "    writer = csv.writer(csvfile)\n",
    "    for key in directories:\n",
    "        writer.writerow([key] + directories[key])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Compile name, size, and extension for file 2\n",
    "path = []\n",
    "name = []\n",
    "size = []\n",
    "ext = []\n",
    "for root, dirs, files in os.walk(startpath):\n",
    "    # Add root as directory (ignore dirs, as they will all be roots)\n",
    "    path.append(root)\n",
    "    name.append(os.path.basename(root))\n",
    "    size.append(None)\n",
    "    ext.append(None)\n",
    "    # Add files individually\n",
    "    for file in files:\n",
    "        filename, file_extension = os.path.splitext(file)\n",
    "        file = root +'/'+ file\n",
    "        path.append(file)\n",
    "        name.append(filename)\n",
    "        size.append(os.stat(file).st_size)\n",
    "        ext.append(file_extension)\n",
    "\n",
    "# Export file2 to csv with pandas\n",
    "import pandas as pd\n",
    "nodes = pd.DataFrame({'Path':path, 'Name':name, 'Size':size, 'Type':ext})\n",
    "nodes.to_csv('NodeTypes_wPaths.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Path</th>\n",
       "      <th>Name</th>\n",
       "      <th>Size</th>\n",
       "      <th>Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PAPER_DATA</td>\n",
       "      <td>PAPER_DATA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype</td>\n",
       "      <td>Gene.expression.by.SNP.genotype</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/IBD...</td>\n",
       "      <td>IBD.GPS_SNP_HLA.subtypes_coefficients_with_ris...</td>\n",
       "      <td>2.765800e+04</td>\n",
       "      <td>.xlsx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/sco...</td>\n",
       "      <td>score_density</td>\n",
       "      <td>2.276770e+05</td>\n",
       "      <td>.png</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/CER...</td>\n",
       "      <td>CERTIFI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/CER...</td>\n",
       "      <td>genotype_EAF</td>\n",
       "      <td>8.259400e+04</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/CER...</td>\n",
       "      <td>stats</td>\n",
       "      <td>3.299733e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/MSH</td>\n",
       "      <td>MSH</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/MSH...</td>\n",
       "      <td>genotype_EAF</td>\n",
       "      <td>2.171800e+04</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/MSH...</td>\n",
       "      <td>stats</td>\n",
       "      <td>2.367450e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/RISK</td>\n",
       "      <td>RISK</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/RIS...</td>\n",
       "      <td>genotype_EAF</td>\n",
       "      <td>2.224900e+04</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>PAPER_DATA/Gene.expression.by.SNP.genotype/RIS...</td>\n",
       "      <td>stats</td>\n",
       "      <td>2.309930e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>PAPER_DATA/IBD Networks</td>\n",
       "      <td>IBD Networks</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks</td>\n",
       "      <td>Co-expressionnetworks</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>Celltype_Enrichment</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_ileum_filtered-vs-abc-cell-gpl5...</td>\n",
       "      <td>7.822530e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_RISK-filtered-vs-abc-tis-gpl570-sign...</td>\n",
       "      <td>1.060932e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_colon_filtered-vs-abc-cell-gpl5...</td>\n",
       "      <td>8.063090e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_RISK-filtered-vs-abc-cell-gpl570-sig...</td>\n",
       "      <td>1.046124e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_ALL_MSH2_filtered-vs-abc-tis-gpl570-...</td>\n",
       "      <td>8.124080e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570...</td>\n",
       "      <td>2.352947e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_T26-vs-abc-tis-gpl570-signatures-fil...</td>\n",
       "      <td>2.275442e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_colon_filtered-vs-abc-cell-gpl5...</td>\n",
       "      <td>2.323958e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_ileum_filtered-vs-abc-tis-gpl57...</td>\n",
       "      <td>7.498670e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_T26-vs-abc-cell-gpl570-signatures-fi...</td>\n",
       "      <td>2.172606e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_RISK-filtered-vs-abc-tis-gpl570-sign...</td>\n",
       "      <td>2.861685e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570...</td>\n",
       "      <td>8.327720e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_colon_filtered-vs-abc-tis-gpl57...</td>\n",
       "      <td>7.778450e+05</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>PAPER_DATA/IBD Networks/Co-expressionnetworks/...</td>\n",
       "      <td>intersect_MSH2_ileum_filtered-vs-abc-cell-gpl5...</td>\n",
       "      <td>2.288802e+06</td>\n",
       "      <td>.xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.952700e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.950600e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.952000e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>CD4Generate overlap between cres-gwas overlaps...</td>\n",
       "      <td>1.589800e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>NKcellsgenerate overlap between cres-gwas over...</td>\n",
       "      <td>1.589200e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>myeloid generate overlap between cres-gwas ove...</td>\n",
       "      <td>1.590600e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>generate overlap between cres-gwas overlaps an...</td>\n",
       "      <td>1.589000e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>link eqtl and gwas snps to ibd genes (1)</td>\n",
       "      <td>8.153000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>CD8Tcell.generate overlap between cres-gwas ov...</td>\n",
       "      <td>1.589300e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.952700e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.950600e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Assemble CRE and GWAS sets of interest and gen...</td>\n",
       "      <td>1.949700e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...</td>\n",
       "      <td>Digestive-Assemble CRE and GWAS sets of intere...</td>\n",
       "      <td>1.952000e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>PAPER_DATA/Code/CREenrich</td>\n",
       "      <td>CREenrich</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>PAPER_DATA/Code/CREenrich/CES.R.txt</td>\n",
       "      <td>CES.R</td>\n",
       "      <td>6.623000e+03</td>\n",
       "      <td>.txt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>PAPER_DATA/Code/CREenrich/db.Rdata</td>\n",
       "      <td>db</td>\n",
       "      <td>1.789333e+10</td>\n",
       "      <td>.Rdata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>PAPER_DATA/Code/CREenrich/.Rhistory</td>\n",
       "      <td>.Rhistory</td>\n",
       "      <td>4.100000e+01</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>PAPER_DATA/Code/CREenrich/CRE.tsv</td>\n",
       "      <td>CRE</td>\n",
       "      <td>1.083441e+06</td>\n",
       "      <td>.tsv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>PAPER_DATA/Code/TF</td>\n",
       "      <td>TF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>PAPER_DATA/Code/TF/main_TFA.R</td>\n",
       "      <td>main_TFA</td>\n",
       "      <td>3.663000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>PAPER_DATA/Code/TF/Run_InferTFA_Public.R</td>\n",
       "      <td>Run_InferTFA_Public</td>\n",
       "      <td>1.757000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>PAPER_DATA/Code/TF/GetOptimalWindowandActivity...</td>\n",
       "      <td>GetOptimalWindowandActivity_Core_Public</td>\n",
       "      <td>9.380000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>PAPER_DATA/Code/analysis</td>\n",
       "      <td>analysis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>PAPER_DATA/Code/Network</td>\n",
       "      <td>Network</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>PAPER_DATA/Code/Network/codeforenrichmenttable.R</td>\n",
       "      <td>codeforenrichmenttable</td>\n",
       "      <td>6.890000e+02</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>PAPER_DATA/Code/Network/co-expression network</td>\n",
       "      <td>co-expression network</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>PAPER_DATA/Code/Network/co-expression network/...</td>\n",
       "      <td>RUN_KDA</td>\n",
       "      <td>3.412000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>PAPER_DATA/Code/Network/co-expression network/...</td>\n",
       "      <td>Curate_Signature</td>\n",
       "      <td>1.487000e+03</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>PAPER_DATA/Code/Network/co-expression network/...</td>\n",
       "      <td>DEG.curated</td>\n",
       "      <td>8.357400e+04</td>\n",
       "      <td>.RData</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>PAPER_DATA/Code/Network/co-expression network/...</td>\n",
       "      <td>R_functions</td>\n",
       "      <td>1.446200e+04</td>\n",
       "      <td>.R</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>327 rows Ã— 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  Path  \\\n",
       "0                                           PAPER_DATA   \n",
       "1           PAPER_DATA/Gene.expression.by.SNP.genotype   \n",
       "2    PAPER_DATA/Gene.expression.by.SNP.genotype/IBD...   \n",
       "3    PAPER_DATA/Gene.expression.by.SNP.genotype/sco...   \n",
       "4    PAPER_DATA/Gene.expression.by.SNP.genotype/CER...   \n",
       "5    PAPER_DATA/Gene.expression.by.SNP.genotype/CER...   \n",
       "6    PAPER_DATA/Gene.expression.by.SNP.genotype/CER...   \n",
       "7       PAPER_DATA/Gene.expression.by.SNP.genotype/MSH   \n",
       "8    PAPER_DATA/Gene.expression.by.SNP.genotype/MSH...   \n",
       "9    PAPER_DATA/Gene.expression.by.SNP.genotype/MSH...   \n",
       "10     PAPER_DATA/Gene.expression.by.SNP.genotype/RISK   \n",
       "11   PAPER_DATA/Gene.expression.by.SNP.genotype/RIS...   \n",
       "12   PAPER_DATA/Gene.expression.by.SNP.genotype/RIS...   \n",
       "13                             PAPER_DATA/IBD Networks   \n",
       "14       PAPER_DATA/IBD Networks/Co-expressionnetworks   \n",
       "15   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "16   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "17   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "18   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "19   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "20   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "21   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "22   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "23   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "24   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "25   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "26   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "27   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "28   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "29   PAPER_DATA/IBD Networks/Co-expressionnetworks/...   \n",
       "..                                                 ...   \n",
       "297  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "298  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "299  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "300  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "301  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "302  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "303  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "304  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "305  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "306  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "307  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "308  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "309  PAPER_DATA/Code/GWASmapandIAMKDG/GWASmapandenr...   \n",
       "310                          PAPER_DATA/Code/CREenrich   \n",
       "311                PAPER_DATA/Code/CREenrich/CES.R.txt   \n",
       "312                 PAPER_DATA/Code/CREenrich/db.Rdata   \n",
       "313                PAPER_DATA/Code/CREenrich/.Rhistory   \n",
       "314                  PAPER_DATA/Code/CREenrich/CRE.tsv   \n",
       "315                                 PAPER_DATA/Code/TF   \n",
       "316                      PAPER_DATA/Code/TF/main_TFA.R   \n",
       "317           PAPER_DATA/Code/TF/Run_InferTFA_Public.R   \n",
       "318  PAPER_DATA/Code/TF/GetOptimalWindowandActivity...   \n",
       "319                           PAPER_DATA/Code/analysis   \n",
       "320                            PAPER_DATA/Code/Network   \n",
       "321   PAPER_DATA/Code/Network/codeforenrichmenttable.R   \n",
       "322      PAPER_DATA/Code/Network/co-expression network   \n",
       "323  PAPER_DATA/Code/Network/co-expression network/...   \n",
       "324  PAPER_DATA/Code/Network/co-expression network/...   \n",
       "325  PAPER_DATA/Code/Network/co-expression network/...   \n",
       "326  PAPER_DATA/Code/Network/co-expression network/...   \n",
       "\n",
       "                                                  Name          Size    Type  \n",
       "0                                           PAPER_DATA           NaN    None  \n",
       "1                      Gene.expression.by.SNP.genotype           NaN    None  \n",
       "2    IBD.GPS_SNP_HLA.subtypes_coefficients_with_ris...  2.765800e+04   .xlsx  \n",
       "3                                        score_density  2.276770e+05    .png  \n",
       "4                                              CERTIFI           NaN    None  \n",
       "5                                         genotype_EAF  8.259400e+04    .xls  \n",
       "6                                                stats  3.299733e+06    .xls  \n",
       "7                                                  MSH           NaN    None  \n",
       "8                                         genotype_EAF  2.171800e+04    .xls  \n",
       "9                                                stats  2.367450e+05    .xls  \n",
       "10                                                RISK           NaN    None  \n",
       "11                                        genotype_EAF  2.224900e+04    .xls  \n",
       "12                                               stats  2.309930e+05    .xls  \n",
       "13                                        IBD Networks           NaN    None  \n",
       "14                               Co-expressionnetworks           NaN    None  \n",
       "15                                 Celltype_Enrichment           NaN    None  \n",
       "16   intersect_MSH2_ileum_filtered-vs-abc-cell-gpl5...  7.822530e+05    .xls  \n",
       "17   intersect_RISK-filtered-vs-abc-tis-gpl570-sign...  1.060932e+06    .xls  \n",
       "18   intersect_MSH2_colon_filtered-vs-abc-cell-gpl5...  8.063090e+05    .xls  \n",
       "19   intersect_RISK-filtered-vs-abc-cell-gpl570-sig...  1.046124e+06    .xls  \n",
       "20   intersect_ALL_MSH2_filtered-vs-abc-tis-gpl570-...  8.124080e+05    .xls  \n",
       "21   intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570...  2.352947e+06    .xls  \n",
       "22   intersect_T26-vs-abc-tis-gpl570-signatures-fil...  2.275442e+06    .xls  \n",
       "23   intersect_MSH2_colon_filtered-vs-abc-cell-gpl5...  2.323958e+06    .xls  \n",
       "24   intersect_MSH2_ileum_filtered-vs-abc-tis-gpl57...  7.498670e+05    .xls  \n",
       "25   intersect_T26-vs-abc-cell-gpl570-signatures-fi...  2.172606e+06    .xls  \n",
       "26   intersect_RISK-filtered-vs-abc-tis-gpl570-sign...  2.861685e+06    .xls  \n",
       "27   intersect_ALL_MSH2_filtered-vs-abc-cell-gpl570...  8.327720e+05    .xls  \n",
       "28   intersect_MSH2_colon_filtered-vs-abc-tis-gpl57...  7.778450e+05    .xls  \n",
       "29   intersect_MSH2_ileum_filtered-vs-abc-cell-gpl5...  2.288802e+06    .xls  \n",
       "..                                                 ...           ...     ...  \n",
       "297  Assemble CRE and GWAS sets of interest and gen...  1.952700e+04      .R  \n",
       "298  Assemble CRE and GWAS sets of interest and gen...  1.950600e+04      .R  \n",
       "299  Assemble CRE and GWAS sets of interest and gen...  1.952000e+04      .R  \n",
       "300  CD4Generate overlap between cres-gwas overlaps...  1.589800e+04      .R  \n",
       "301  NKcellsgenerate overlap between cres-gwas over...  1.589200e+04      .R  \n",
       "302  myeloid generate overlap between cres-gwas ove...  1.590600e+04      .R  \n",
       "303  generate overlap between cres-gwas overlaps an...  1.589000e+04      .R  \n",
       "304           link eqtl and gwas snps to ibd genes (1)  8.153000e+03      .R  \n",
       "305  CD8Tcell.generate overlap between cres-gwas ov...  1.589300e+04      .R  \n",
       "306  Assemble CRE and GWAS sets of interest and gen...  1.952700e+04      .R  \n",
       "307  Assemble CRE and GWAS sets of interest and gen...  1.950600e+04      .R  \n",
       "308  Assemble CRE and GWAS sets of interest and gen...  1.949700e+04      .R  \n",
       "309  Digestive-Assemble CRE and GWAS sets of intere...  1.952000e+04      .R  \n",
       "310                                          CREenrich           NaN    None  \n",
       "311                                              CES.R  6.623000e+03    .txt  \n",
       "312                                                 db  1.789333e+10  .Rdata  \n",
       "313                                          .Rhistory  4.100000e+01          \n",
       "314                                                CRE  1.083441e+06    .tsv  \n",
       "315                                                 TF           NaN    None  \n",
       "316                                           main_TFA  3.663000e+03      .R  \n",
       "317                                Run_InferTFA_Public  1.757000e+03      .R  \n",
       "318            GetOptimalWindowandActivity_Core_Public  9.380000e+03      .R  \n",
       "319                                           analysis           NaN    None  \n",
       "320                                            Network           NaN    None  \n",
       "321                             codeforenrichmenttable  6.890000e+02      .R  \n",
       "322                              co-expression network           NaN    None  \n",
       "323                                            RUN_KDA  3.412000e+03      .R  \n",
       "324                                   Curate_Signature  1.487000e+03      .R  \n",
       "325                                        DEG.curated  8.357400e+04  .RData  \n",
       "326                                        R_functions  1.446200e+04      .R  \n",
       "\n",
       "[327 rows x 4 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nodes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
